singer-io/tap-bing-ads
# TDL-15251: Add required fields for age_gender_audience_report

# Description of change
- Updated the required columns dictionary by adding required fields for AgeGederAudienceReport

# Manual QA steps
 - Run the discovery mode and check that the catalog has the fields AccountName, AdGroupName, AgeGroup, Gender marked as inclusion automatic or not.

# Risks
 - 
 
# Rollback steps
 - revert this branch
